tests/pages/firefox/base.py

```diff
@@ -12,8 +12,11 @@
 class ScrollElementIntoView(object):
 
     def scroll_element_into_view(self, strategy, locator):
+        # scroll elements so they are not beneath the header
+        page = self.page if isinstance(self, Region) else self
+        offset = {'x': 0, 'y': -page.family_navigation.root.size['height']}
```

Comment:
This allows it to be inconsistent - it will scroll based on the height of the header. If the header is taller in some browsers then we'll scroll further to accommodate it.